_sass/pages/_docs.scss

```diff
@@ -1,72 +1,112 @@
 // Docs layout specific
 .docs {
-  
-  *[id]:before { 
-    display: block; 
-    content: " "; 
-    margin-top: -1em;
-    height: 1em;
-    visibility: hidden; 
-  }
-
   .doc-header {
-    padding-bottom: 30px;
-    padding-top: 30px;
-  }
-  
-  .doc-description {
-    color: $dark-gray;
-    display: block;
-    margin: 0 0 40px;
-
-    p { 
-      font-size: 18px;
-      font-weight: 400;
+    padding-top: 64px;
+    padding-bottom: 24px;
+
+    @media (max-width: $phone-xlarge) {
+      padding-top: 48px;
     }
   }
 
-  .use-description {
-    color: $gray;
-    display: block;
-    margin: 20px 0;
-
-    p { 
-      font-size: 15px;
-      font-weight: 300;
+  .info-section {
+    margin: 0 10px 0;
+    @media (max-width: $phone-xlarge){
+      margin-top: 16px;
     }
   }
 
   .lang-card-selector {
-    padding: 0 0 30px;
+    padding: 0 0 88px;
     width: 100%;
+
+    @media(max-width: $phone-xlarge){
+      padding-bottom: 24px;
+    }
   }
 
   .lang-card {
     display: inline-block;
     margin: 0 1% 30px;
-    text-align: center;
-    vertical-align: top;
+    padding: 24px 24px 32px;
     width: 30%;
+    background-image: linear-gradient(-180deg, $white 0%, #FAFAFA 100%);
+    box-shadow: 0 4px 8px 0 rgba($footerColor, .12), 0 -1px 2px 0 rgba($footerColor, .05);
+    border-radius: 4px;
 
-    @media (max-width: $tablet) {
-      margin: 20px 1%;
-      padding: 6px 10px;
+    @media (max-width: $desktop-small) {
       width: 47%;
     }
+
+    @media (max-width: $phone-xlarge) {
+      width: 98%;
+    }
     
     .lang-card-title {
-      background: $light-gray;
       color: $dark-gray;
-      margin-bottom: 4px;
-      padding: 8px;
-      text-align: left;
+      margin-bottom: 16px;
+      font-size: 1rem;
+      font-weight: 700;
+    }
+
+    @mixin lang-icon($image-url) {
+      &::before {
```

Comment:
It would be better to move common properties outside of the @mixin. Because each selector will use same set of styles.